packages/xo-web/src/xo-app/logs/backup-ng-logs.js

```diff
@@ -113,7 +118,25 @@ const showTasks = log =>
       {_('jobModalTitle', { job: log.jobId.slice(4, 8) })}{' '}
       <span style={{ fontSize: '0.5em' }} className='text-muted'>
         {log.id}
-      </span>
+      </span>{' '}
+      {CAN_REPORT_BUG &&
+        log.status !== 'success' &&
+        log.status !== 'pending' && (
+          <ButtonGroup>
+            <Tooltip content={_('copyToClipboard')}>
+              <CopyToClipboard text={JSON.stringify(log, null, 2)}>
```

Comment:
Do you think it's a good idea, performant-wise, to format to JSON twice at every render?

Reply:
It's an error, i will fix it.